viewer/packages/camera-manager/src/Flexible/FlexibleControlsOptions.ts

```diff
@@ -117,6 +118,13 @@ export class FlexibleControlsOptions {
     }
   }
 
+  public getKeyboardSpeed(shift: boolean): number {
+    if (!shift) {
```

Comment:
If I understood this correctly, the keyboard speed will be enabled only when the user is pressing shift?
It was requested? I thought that the user only needs to set the speed from the dropdown and just use it. If the user has to keep pressing the shift while wanting to keep the movement faster, is it not be annoying?

Reply:
No. Look at the code.  The control I have made changes this.keyboardSpeed. This used when moving by keyboard. So the user is able to change that. 

If shift is pressed, the speed will be multiplied by  this.keyboardFastMoveFactor.

Reply:
Ah, yeah, right. So we already have a move factor when pressing shift and it is already there from a long time, way before this keyboard speed... you're right.
All good!